### DISCUSSION TITLE
Profile information overlaps.

### DISCUSSION THREAD

**Discussion by yusasarisoy** [community/community#63106] [Mobile] [answered]
![image](https://github.com/community/community/assets/30107357/9cff8df0-e83b-49e2-b250-d44df81633e3)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.124.0 (105848740)
Device: iPhone 7 (iOS 15.7.8)
</details>

**Comment by mattruggio** [comment, marked as answer]
Hey @yusasarisoy, thank you for reporting this issue! [Another community discussion](https://github.com/orgs/community/discussions/63101) came in regarding the exact same bug. This bug is specific to running the GitHub iOS application on iOS 15 (iOS 16 looks to function correctly). The mobile team has acknowledged this specific iOS 15 bug and will be working on putting out a fix in a future release.

**Reply by yusasarisoy** [reply]
Thank you for the reply.
